### PR TITLE
3952 math table spacing

### PR DESCRIPTION
**Pull request by LostMojo**
#3952

### CODE REVIEW

**Comment by LostMojo** [comment]
Screen Shot 2022-01-04 at 3 19 21 PM

File: styles/designs/corn/parts/_table-components.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@ $Table__Container: (
     _subselector: "",
     _properties: (
         margin-bottom: table-v-spacing(2),
+        margin-top: table-v-spacing(1),
```
Comment (wilkus27) [inline styles/designs/corn/parts/_table-components.scss:6]: I think we should be more specific :thinking: There is no space between table and text in certain situations. This change is added to all tables. If table be after element with bottom padding (e.g para in module) there will be double space there (bottom padding of para and top padding of table).

**Review by wilkus27 — CHANGES_REQUESTED** [review]

File: styles/designs/corn/parts/_table-components.scss (hunk 1 above)
Comment (ajluk) [reply styles/designs/corn/parts/_table-components.scss:6]: Natalia has already written what I'm concerned about as well. Maybe we can tag such tables with specific class in content and add top margins only to them? :thinking:

Comment (LostMojo) [reply styles/designs/corn/parts/_table-components.scss]: So I've changed it to only take effect in tables in lists-- but due to how the html is set up, there's not a way to always catch when there's not text above a list-bound table-- I think it should be fine.

Comment (LostMojo) [reply styles/designs/corn/parts/_table-components.scss]: I've limited created a different shape to only target tables in lists-- this should cover most of our bases, and Larissa says we probably don't have to worry too much about the edge case of a list item starting with a table!

**Review by wilkus27 — APPROVED** [review]

**Pull request merged**